sdk/core/azure-core-amqp/inc/azure/core/amqp/internal/message_sender.hpp

```diff
@@ -9,9 +9,9 @@
 #include "connection.hpp"
 #include "link.hpp"
 #include "models/amqp_error.hpp"
-#include "models/amqp_message.hpp"
-#include "models/amqp_value.hpp"
 
+#include <azure/core/amqp/models/amqp_message.hpp>
+#include <azure/core/amqp/models/amqp_value.hpp>
```

Comment:
`<>` => `""`

Reply:
FWIW, it's not necessary for you to call out every instance of the issue in the PR. It just makes more work for both you and I.